src/init.lua

```diff
@@ -1979,7 +1981,7 @@ do -- Search stuff
 	searchBox.Changed:Connect(onChanged)
 	searchBox.FocusLost:Connect(focusLost)
 
-	BackpackScript.StateChanged.Event:Connect(function(isNowOpen: boolean)
+	BackpackScript.StateChanged:Connect(function(isNowOpen: boolean)
```

Comment:
`BackpackScript.StateChanged` returns the type `BindableEvent` on line 2148 so we would need to correct. This could cause a warning here.